crates/matrix-sdk/src/room/futures.rs

```diff
@@ -172,17 +174,17 @@ impl<'a> IntoFuture for SendRawMessageLikeEvent<'a> {
             Span::current().record("transaction_id", tracing::field::debug(&txn_id));
 
             #[cfg(not(feature = "e2e-encryption"))]
-            debug!("Sending plaintext event to room because we don't have encryption support.");
+            trace!("Sending plaintext event to room because we don't have encryption support.");
 
             #[cfg(feature = "e2e-encryption")]
             if room.is_encrypted().await? {
-                Span::current().record("encrypted", true);
+                Span::current().record("encrypted_room", true);
```

Comment:
You can't just rename the field here, this will not record anything now, the span fields need to be renamed here: https://github.com/matrix-org/matrix-rust-sdk/blob/565f97409ed181c713d0c358b10d2d6085ca4404/crates/matrix-sdk/src/room/mod.rs#L1688

Not really a requirement, but I think `is_room_encrypted` would be even more clear, feel free to disagree.

Reply:
Ah, I was looking where the field is, thanks.